bundle.yaml

```diff
@@ -14,10 +15,44 @@ applications:
     charm: grafana-k8s
     channel: edge
     scale: 1
+  loki:
+    charm: loki-k8s
+    channel: edge
+    scale: 1
 
 relations:
-- - prometheus
-  - alertmanager
-- - grafana
-  - prometheus
+- - prometheus:alertmanager
+  - alertmanager:alerting
+- - grafana:grafana-source
+  - prometheus:grafana-source
+#- - grafana:grafana-source
```

Comment:
@sed-i why commenting this out? https://github.com/canonical/loki-operator/blob/b594afe343cba3a44c7fffe0c2bb11063400470a/metadata.yaml#L43